src/lib/phdi-building-blocks/phdi_building_blocks/fhir.py

```diff
@@ -231,12 +284,16 @@ def __export_from_fhir_server_poll_call(
 
 
 def export_from_fhir_server_poll(
-    poll_url: str, access_token: str, poll_step: float = 30, poll_timeout: float = 300
+    poll_url: str,
+    cred_manager: AzureFhirserverCredentialManager,
+    poll_step: float = 30,
```

Comment:
I think these two numbers should also be constants to be clear on what the magic numbers are.